Repositories/CrudRepository.cs

```diff
@@ -40,14 +41,15 @@ public T Find(int key)
         public IEnumerable<T> GetAll()
         {
             var items = GetQuery(Context);
-            return items.ToList();
+            return items.Where(i => EF.Property<bool>(i, "IsDeleted") == false).ToList();
```

Comment:
This is a good candidate to move to extension method like "items.NotDeleted().ToList()" and use it everywhere where we need to filter out deleted items.